package.json

```diff
@@ -1,36 +1,10 @@
 {
   "name": "analyze_api_ui",
-  "version": "7.6.2",
-  "description": "UI for elasticsearch analyze API",
-  "main": "index.js",
-  "kibana": {
-    "version": "7.6.2",
-    "templateVersion": "1.0.0"
-  },
+  "version": "7.16.2",
```

Comment:
This should be updated?

Reply:
It's OK. I'll update after merging this PR